pylib/spinnaker/import_export.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/python
+
+'''
+
+# Spinnaker Import/Export Tool
+
+Python cli tool that imports and exports the following Spinnaker items:
+
+* Applications
+  * Pipelines
+  * Deployment Strategies
+  * Task Logs
```

Comment:
Since task executions are persisted in redis, task history won't be captured. Same for pipeline execution history.